compiler/src/iree/compiler/Dialect/HAL/IR/HALOps.cpp

```diff
@@ -1074,14 +1088,16 @@ LogicalResult InterfaceBindingSubspanOp::verify() {
 }
 
 // TODO(benvanik): share with align op folder and analysis.
-// May need an interface for querying the alignment from ops that can carry it.
```

Comment:
Please revert the comment changes in here. I think you may have run a formatter with narrower widths? (sometimes can happen if code becomes unbalanced and the file is saved - clang-format will happily indent everything then reflow comments but not fix them when the balancing is corrected)